homeassistant/components/hive/sensor.py

```diff
@@ -77,7 +114,7 @@ def __init__(
         self,
         hive: Hive,
         hive_device: dict[str, Any],
-        entity_description: SensorEntityDescription,
+        entity_description: HiveSensorEntityDescription,
```

Comment:
Also please overwrite the type outside of the constructor

Reply:
Hmmm.....I'm unsure what you mean by _the type outside the constructor_.  I can't really see which other types would need to be updated?

Reply:
You mean like this: `entity_description: HiveSensorEntityDescription`?